File: AllSharpProject/TestLibrary/Client/ClientQuestionAnswer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace TestLibrary.ClientEdit
{
    /// <summary>
    /// ответ клиента на вопрос
    /// </summary>
    [XmlRoot("Answer")]
    public class ClientQuestionAnswer : TestLibrary.Helpers.NotificationObject
    {
        protected MediaData data;

        /// <summary>
        /// ответ на вопрос
        /// </summary>
        [XmlElement("MediaData")]
        public MediaData Data
        {
            get
            {
                if (data == null)
                {
                    return new MediaData();
                }
                else
                {
                    return data;
                }
            }
            set
            {
                this.data = value;
                RaisePropertyChanged(() => this.Data);
            }
        }
    }
}

File: TestClient/src/TestLibrary/Client/ClientQuestionAnswer.java
package TestLibrary.Client;

import TestLibrary.MediaData;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;


/**
 * Created by Pit on 13.06.13.
 */
@Root(name = "Answer")
public class ClientQuestionAnswer
{
    @Element(name = "MediaData")
    public MediaData Data  = new MediaData();
}
